src/pages/store/StoreEdit.tsx

```diff
@@ -183,119 +185,111 @@ export default function StoreEdit() {
-            />
-          </div>
-        </div>
-
-        <div className="mb-20 flex flex-col gap-20 md:mb-24 md:gap-24">
-          <div className="flex flex-col gap-20 md:flex-row md:gap-20">
-            <div className="flex flex-col gap-8 md:max-w-472 md:basis-1/2">
-              <label className="text-body1/26 font-regular text-black">
-                주소*
-              </label>
-              <Dropdown<Address1>
-                options={ADDRESS_OPTIONS}
-                selected={edit.address1}
-                setSelect={(value) =>
-                  setEdit((prev) => ({ ...prev, address1: value as Address1 }))
-                }
-                variant="form"
+        <form
+          onSubmit={handleSubmit}
+          className="flex flex-col gap-24 md:gap-32"
+        >
+          <div className="flex flex-col gap-20 md:gap-24">
+            <div className="grid grid-cols-1 gap-20 md:grid-cols-2 md:gap-y-24">
+              <Input
+                label="가게 이름*"
+                value={edit.name}
+                onChange={handleChange('name')}
               />
-            </div>
-            <div className="md:max-w-472 md:basis-1/2">
+              <div className="flex flex-col gap-8">
+                <label className="text-body1/26 font-regular text-black">
+                  분류*
+                </label>
+                <Dropdown<Category>
+                  options={CATEGORY_OPTIONS}
+                  selected={edit.category}
+                  setSelect={(value) =>
+                    setEdit((prev) => ({
+                      ...prev,
+                      category: value as Category,
+                    }))
+                  }
+                  variant="form"
+                />
+              </div>
+              <div className="flex flex-col gap-8">
+                <label className="text-body1/26 font-regular text-black">
+                  주소*
+                </label>
+                <Dropdown<Address1>
+                  options={ADDRESS_OPTIONS}
+                  selected={edit.address1}
+                  setSelect={(value) =>
+                    setEdit((prev) => ({
+                      ...prev,
+                      address1: value as Address1,
+                    }))
+                  }
+                  variant="form"
+                />
+              </div>
               <Input
                 label="상세 주소*"
                 value={edit.address2}
                 onChange={handleChange('address2')}
               />
-            </div>
-          </div>
-          <div className="flex flex-col md:flex-row md:items-center md:gap-20">
-            <div className="md:max-w-472 md:basis-1/2">
               <Input
                 label="기본 시급*"
                 value={
                   edit.originalHourlyPay === 0
                     ? ''
-                    : formatNumber(String(edit.originalHourlyPay))
+                    : Number(edit.originalHourlyPay).toLocaleString()
```

Comment:
```suggestion
                    : edit.originalHourlyPay.toLocaleString()
```
💬 `originalHourlyPay`가 원래 `number`형이라 이대로 써도 될 것 같습니다~

Reply:
넵 알겠습니다!